models/balancer/ethereum/balancer_ethereum_sources.yml

```diff
@@ -1,25 +1,51 @@
 version: 2
 
 sources:
+  - name: balancer_v2_ethereum
+    description: >
+      Decoded tables related to Balancer v2, an automated portfolio manager and trading platform.
+    tables:
+      - name: Vault_evt_PoolRegistered
+        description: >
+          Decoded table of registered pools on the Balancer Vault contract.
+        loaded_at_field: evt_block_time
+        columns:
+          - &contract_address
+            name: contract_address
+            description: 'Ethereum address for the Balancer Vault contract used in transaction'
+          - &evt_tx_hash
+            name: evt_tx_hash
+            description: 'Primary key of the transaction'
+          - &evt_index
+            name: evt_index
+            description: 'Index value of the transaction'
+          - &evt_block_time
+            name: evt_block_time
+            description: 'Timestamp for block event time in UTC'
+          - &evt_block_number
+            name: evt_block_number
+            description: 'Block number which processed the unique transaction hash'
+          - name: poolAddress
+            description: 'Ethereum address for the liquidity pool used in transaction'
+          - name: poolId
+            description: 'Unique encoded identifier that refers to each pool'
+          - name: specialization
+            description: 'Pool specialization'
+
   - name: balancer_v1_ethereum
     description: >
-      Decoded tables related to Balancer, an automated portfolio manager and trading platform.
+      Decoded tables related to Balancer v1, an automated portfolio manager and trading platform.
     tables:
       - name: BFactory_evt_LOG_NEW_POOL
         description: >
           Decoded table related to the Balancer BFactory contract.
         loaded_at_field: evt_block_time
         columns:
-          - name: contract_address
-            description: 'Balancer BFactory contract address'
-          - name: evt_tx_hash
-            description: 'Transaction hash of the event'
-          - name: evt_index
-            description: 'Event index'
-          - name: evt_block_time
-            description: 'Timestamp for block event time in UTC'
-          - name: evt_block_number
-            description: 'Event block number'
+          - *contract_address
```

Comment:
would this lead the `contract_address` from V1 factory to have the description from V2, "Ethereum address for the Balancer Vault contract used in transaction"?

Reply:
Yes, you are correct, the `contract_address` would have the same description for both tables.

I have pushed a fix so that each table has a different description for the `contract_address` column.